contrib/osx/notes_for_arm.md

```diff
@@ -0,0 +1,56 @@
+# Notes on building Electrum on Apple M1 OSX (ARM)
+
+Development version (git clone)
+
+1. Check out the code from GitHub:
+
+```
+$ git clone https://github.com/spesmilo/electrum.git
+$ cd electrum
+$ git submodule update --init
+```
+
+Run install (this should install dependencies):
+```
+python3 -m pip install --user -e .
+```
+
+2. pycryptodomex and cryptography and will need to be manually installed:
+
+```
+$ pip install pycryptodomex
+$ pip install cryptography
+```
```

Comment:
You don't need both, do you?
Either one should be enough; they are alternatives to each other.
Assuming that's so, we prefer `cryptography` (or at least let's be consistent with the top-level README.rst).

Reply:
Ok lets just make it `cryptography`